BDArmory/BDArmory.csproj

```diff
@@ -58,72 +58,174 @@
   <ItemGroup>
     <Reference Include="Assembly-CSharp, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\_LocalDev\KSPRefs\Assembly-CSharp.dll</HintPath>
+      <HintPath>..\..\..\..\..\..\..\..\Program Files (x86)\Steam\steamapps\common\Kerbal Space Program\KSP_x64_Data\Managed\Assembly-CSharp.dll</HintPath>
```

Comment:
Any changes to the `BDArmory.csproj` file need to respect the `..\..\_LocalDev\KSPRefs` path so that it is consistent between build environments. On Linux, `KSPRefs` can be a symlink to `Kerbal Space Program\KSP_Data\Managed` (Linux installs of KSP use `KSP_Data` instead of `KSP_x64_Data` for some reason). This may also be possible on Windows, otherwise you can copy the DLLs in the `Managed` folder to the `KSPRefs` folder.

Also, you appear to be adding a number of extra reference DLLs to the `BDArmory.csproj` file, but I don't see anywhere that you're using them, so this is an unnecessary change.